scripts/release/docs_update_experiments.sh

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
```

Comment:
BTW didn't we agree on moving to Go with new scripts?

Reply:
That's fair, I began prototyping this as Bash (because Go isn't great for prototyping CLI command execution) and then didn't want to rewrite it 😅. But now that we know what needs to be done, doing so should be fairly easy.